_posts/2017-08-16-node_starter_guide.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Starters Guide to Node.js"
+date: 2017-08-16 00:00:00 +530
+author: Hritvik Taneja
+category:
+tags:
+- node
+- Node.js
+- npm
+---
+## `console.log('Hello World')`
+
+I assume that most of you are here to learn about Node.js. So, lets get started!!
+
+## What is Node.js ?
+
+**If you go by docs:** [Node.js®](https://nodejs.org/en/) is a JavaScript runtime built on [Chrome's V8 JavaScript engine](https://developers.google.com/v8/). Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, [npm](https://www.npmjs.com/), is the largest ecosystem of open source libraries in the world.
+
+In simple words Node.js is server side programming environment which helps us to create APIs which can be consumed by Mobile devices, Websites etc..
```

Comment:
Again, I think some explanation of the word "API" etc might be needed. Maybe add a link to one of the articles which have client/server thing explained (since most people won't know that, but here it will be off topic).